### PR TITLE
Lua Script Updates

### PR DESCRIPTION
**Pull request by deadbytefpv**
A few updates for the Lua Script How-to page.

ExpressLRS 3.4.0 brings a few more modes and Receiver options and also deprecates or removes older implementation.

This should be merged AFTER #382

### CODE REVIEW

**Comment by deadbytefpv** [comment]
Hold off merging until AFTER #361 and #360 are merged.. Conflicts could arise after those.

**Comment by deadbytefpv** [comment]
Should be ready for review and subsequent merge.

Thanks!

**Review by wvarty — APPROVED** [review]

**Review by maybenikhil — APPROVED** [review]

**Pull request merged**
